crash: fatal error in soup_session_feature_detach with WebKitGTK+ >= 1.11.91
http://git.eclipse.org/c/platform/eclipse.platform.swt.git/commit/?id=b22a7d19afbe2a3811a0f8aa54c1e85d92c62a2c
https://bugs.eclipse.org/404776
https://bugs.gentoo.org/500570

--- a/src/org/eclipse/swt/browser/WebKit.java
+++ b/src/org/eclipse/swt/browser/WebKit.java
@@ -597,9 +597,13 @@ public void create (Composite parent, int style) {
 	*/
 	long /*int*/ session = WebKitGTK.webkit_get_default_session ();
 	long /*int*/ originalAuth = WebKitGTK.soup_session_get_feature (session, WebKitGTK.webkit_soup_auth_dialog_get_type ());
-	WebKitGTK.soup_session_feature_detach (originalAuth, session);
+	if (originalAuth != 0) {
+		WebKitGTK.soup_session_feature_detach (originalAuth, session);
+	}
 	OS.g_signal_connect (session, WebKitGTK.authenticate, Proc5.getAddress (), webView);
-	WebKitGTK.soup_session_feature_attach (originalAuth, session);
+	if (originalAuth != 0) {
+		WebKitGTK.soup_session_feature_attach (originalAuth, session);
+	}
 
 	/*
 	* Check for proxy values set as documented java properties and update the
